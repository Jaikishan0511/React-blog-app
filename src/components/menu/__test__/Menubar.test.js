import React from "react";
import renderer from "react-test-renderer";

import Menubar from "../index";
describe("Menubar", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Menubar />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
