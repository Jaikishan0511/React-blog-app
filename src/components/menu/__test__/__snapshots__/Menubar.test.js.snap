// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menubar snapshot renders 1`] = `
<div>
  <ul>
    <li>
      list1
    </li>
    <li>
      list2
    </li>
    <li>
      list3
    </li>
  </ul>
</div>
`;
